src/status_im/ui/screens/ens/views.cljs

```diff
@@ -487,35 +495,77 @@
+     [{:style {:color colors/black :text-align :center}}
+      (str "\n@" preferred-name)]]]
+   [react/scroll-view
+    (for [name names]
+      (let [action #(do (re-frame/dispatch [:ens/save-preferred-name name])
+                        (reset! bottom-shown? false))]
+        ^{:key name}
+        [react/touchable-highlight {:on-press action}
+         [react/view {:style {:flex 1 :flex-direction :row :align-items :center :justify-content :center :margin-right 16}}
+          [react/view {:style {:flex 1}}
+           [name-item {:name name :hide-chevron? true :action action}]]
+          [radio/radio (= name preferred-name)]]]))]])
+
+(defn- registered [names {:keys [preferred-name address name public-key] :as account} show? bottom-shown?]
+  [react/scroll-view
+   [react/view {:style {:margin-top 8}}
     [list/big-list-item {:text      (i18n/label :t/ens-add-username)
                          :action-fn #(re-frame/dispatch [:navigate-to :ens-register])
                          :icon      :main-icons/add}]]
-   [react/view {:style {:margin-top 22}}
+   [react/view {:style {:margin-top 22 :margin-bottom 8}}
     [react/text {:style {:color colors/gray :margin-horizontal 16}}
      (i18n/label :t/ens-your-usernames)]
     (if (seq names)
       [react/view {:style {:margin-top 8}}
        (for [name names]
          ^{:key name}
-         [react/view
-          (let [stateofus-username (stateofus/username name)
-                s                  (or stateofus-username name)]
-            [list/big-list-item {:text      s
-                                 :subtext   (when stateofus-username stateofus/domain)
-                                 :action-fn #(re-frame/dispatch [:ens/navigate-to-name name])
-                                 :icon      :main-icons/username}])])]
+         [name-item {:name name :action #(re-frame/dispatch [:ens/navigate-to-name name])}])]
       [react/text {:style {:color colors/gray :font-size 15}}
-       (i18n/label :t/ens-no-usernames)])]])
+       (i18n/label :t/ens-no-usernames)])]
+   [react/view {:style {:padding-top 22 :border-color colors/gray-light :border-top-width 1}}
+    [react/text {:style {:color colors/gray :margin-horizontal 16}}
+     (i18n/label :t/ens-chat-settings)]
+    (when (seq names)
+      [profile.components/settings-item {:label-kw  :ens-primary-username
+                                         :value     preferred-name
+                                         :action-fn #(reset! bottom-shown? true)}])
+    [profile.components/settings-switch-item {:label-kw  :ens-show-username
+                                              :action-fn #(re-frame/dispatch [:ens/switch-show-username])
+                                              :value     show?}]]
+   (let [message {:username name :from public-key :last-in-group? true :display-username? true :display-photo? true
+                  :content {:text "Hey"} :content-type "text/plain" :timestamp-str "9:41 AM"}]
```

Comment:
what are the "Hey" and "9:41 AM" strings, some default values? need localization?

Reply:
Good point. of those two I'd say that 'Hey' that appears in the dummy message should be localised.